WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/IntelliJActivationListener.java

```diff
@@ -5,7 +5,7 @@
 import nl.tudelft.watchdog.intellij.logic.ui.EventManager;
 import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent;
 
-public class IntelliJActivationListener implements ApplicationActivationListener {
+public class IntelliJActivationListener extends ApplicationActivationListener.Adapter {
```

Comment:
This looks really, really strange. No way to Access the adapter? Moreover class is called listener, but is actually an Adapter?